dbx/jvm/v5.2-wn-items.rst

```diff
@@ -9,6 +9,32 @@
   can use a configured FIPS-compliant JCA provider to provide a higher
   level of security.
 
+- Performance improvements due to implementation of native cryptography
+  on all supported platforms. The following list describes the actions
+  needed to implement this improvement depending on your operating
+  system:
+  
+  - **Windows**: Upgrade to your ``mongodb-crypt`` version to v5.2.0.
+
+  - **Mac**: Upgrade to your ``mongodb-crypt`` version to v5.2.0.
+
+  - **Linux**: Install ``libmongocrypt.so`` directly on the file system,
+    instead of using the file that is bundled within the
+    ``mongodb-crypt`` JAR file. View Linux instructions for installing
+    ``libmongocrypt`` in the :manual:`Server manual
+    </core/csfle/reference/libmongocrypt/#linux-installation>`. If you
+    use a package manager to install ``libmongocrypt``, JNA will find it
```

Comment:
Q: Has JNA already been defined on this page/these pages?

Reply:
wrote it out